src/main/java/uk/gov/digital/ho/hocs/workflow/domain/repositories/JsonFileReader.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.digital.ho.hocs.workflow.domain.repositories;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.workflow.application.LogEvent;
+import uk.gov.digital.ho.hocs.workflow.domain.exception.ApplicationExceptions;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Service
+public abstract class JsonFileReader {
```

Comment:
I'm not actually convinced this is a useful helper - we could and probably should role this into the ScreenRepository